docs/source/video_tutorials.rst

```diff
@@ -0,0 +1,39 @@
+Video Tutorials
+===============
+
+Basics of *stk*
+---------------
+
+https://www.youtube.com/watch?v=mPr9D7nCQ84&list=PLIWYdPQ9hLzVngMF8NOkiApMtgc_ZwZgO&index=1
+
+Functional Groups in *stk*
+--------------------------
```

Comment:
Overview of defining functional groups in _stk_. Includes an example of the versatility of _stk_ to unique problems.

Jupyter Notebook is available at https://github.com/andrewtarzia/stk-examples/tree/main/notebooks_from_videos .